cmd/gce-pd-csi-driver/main.go

```diff
@@ -122,7 +124,9 @@ func handle() {
 		if err != nil {
 			klog.Fatalf("Failed to get cloud provider: %v", err)
 		}
-		controllerServer = driver.NewControllerServer(gceDriver, cloudProvider)
+		initialBackoffDuration := time.Duration(*errorBackoffInitialDurationMs) * time.Millisecond
+		maxBackoffDuration := time.Duration(*errorBackoffMaxDurationMs) * time.Microsecond
```

Comment:
@judemars, looks like this should be time.Millisecond.

Reply:
Thanks! Sorry about that! Assigned https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1098 to you.